happybase/pool.py

```diff
@@ -48,10 +48,14 @@ class ConnectionPool(object):
     task of the pool.
 
     :param int size: the maximum number of concurrently open connections
+    :param bool autoconnect: Whether a connection should be created upon
+        instantiation to test connection availability.
+        Note: the `autoconnect` flag of Connection instance inside the pool
+        is always False.
     :param kwargs: keyword arguments passed to
                    :py:class:`happybase.Connection`
     """
-    def __init__(self, size, **kwargs):
+    def __init__(self, size, autoconnect=True, **kwargs):
```

Comment:
can you add a `since` comment (see other docstrings) to make clear when this api was added/changed?